coral/coral/pkg/graphs/resource_models/License.json

```diff
@@ -8761,40 +10071,11 @@
                     "sortorder": 0
                 },
                 {
-                    "alias": "decision_approved_by",
```

Comment:
once again in case this worries you it's actually at line 6186 with the same id's